zerver/models.py

```diff
@@ -1689,6 +1702,7 @@ class UserBaseSettings(models.Model):
         send_stream_typing_notifications=bool,
         web_mark_read_on_scroll_policy=int,
         user_list_style=int,
+        stream_unreads_display=int,
```

Comment:
Have we discussed the field name in "api design"? I think "web_stream_unreads_display_policy" would probably be most appropriate, since this is a web app specific setting, and we've been using `_policy` as a suffix for these integer-format fields.

Reply:
No, we haven't discussed the field name yet, just opened this topic on CZO for it - [Topic Link](https://chat.zulip.org/#narrow/stream/378-api-design/topic/property.20name.20for.20.22show.20unread.20counts.22.20user.20setting.2E).

Although I like the name `web_stream_unreads_display_policy`, I will wait for some feedback from folks on CZO before renaming it.